### PR TITLE
Path correction and Testcase update

### PR DESCRIPTION
**Pull request by LuMAXW3B**
TinyAuthorize::validate() need the correct _config key, trough BC
haven't been updated yet.
Now the TinyAuthorizeTest Class should use the "Plugin::path('TinyAuth')
. 'tests' . DS . 'test_files' . DS" instead of TMP folder.

### CODE REVIEW

**Comment by dereuromark** [comment]
Good work! Thx :+1:

**Comment by LuMAXW3B** [comment]
Ah, each testcase need the filePath beeing set now.

File: tests/TestCase/Auth/TinyAuthorizeTest.php [hunk 1]
```diff
@@ -115,8 +116,8 @@ public function setUp() {
 * = moderator
 INI;
 
-		file_put_contents(TMP . 'acl.ini', $aclData);
-		$this->assertTrue(file_exists(TMP . 'acl.ini'));
+		file_put_contents(Plugin::path('TinyAuth') . 'tests' . DS . 'test_files' . DS . 'acl.ini', $aclData);
```
Comment (dereuromark) [inline tests/TestCase/Auth/TinyAuthorizeTest.php]: Why not keeping TMP?

Comment (LuMAXW3B) [inline tests/TestCase/Auth/TinyAuthorizeTest.php]: Yes you're right, I thought to go analog with the new tests/test_files/auth_allow.ini folder structure but didn't want to change the TinyAuthorizeTest completely. Go like you want.

Comment (dereuromark) [inline tests/TestCase/Auth/TinyAuthorizeTest.php]: Yeah but it does not change the auth_allow file inside tests dir, does it?

Comment (LuMAXW3B) [inline tests/TestCase/Auth/TinyAuthorizeTest.php]: No, it do not change the auth_allow.ini. Yes, a clean testcase paradigm would be good.

**Comment by dereuromark** [comment]
Are you able to provide the changes and make tests pass again?

File: tests/TestCase/Auth/TinyAuthorizeTest.php [hunk 2]
```diff
@@ -8,11 +8,13 @@
...
 use ReflectionClass;
 use TinyAuth\Auth\TinyAuthorize;
+use Cake\Core\Plugin;
 
 /**
  * Test case for TinyAuth Authentication
  */
-class TinyAuthorizeTest extends TestCase {
+class TinyAuthorizeTest extends TestCase
+{
```
Comment (dereuromark) [inline tests/TestCase/Auth/TinyAuthorizeTest.php]: Please keep the psr2r style. Try to keep it noise free.

File: tests/TestCase/Auth/TinyAuthorizeTest.php [hunk 3]
```diff
@@ -8,11 +8,13 @@
 use Cake\TestSuite\TestCase;
 use ReflectionClass;
 use TinyAuth\Auth\TinyAuthorize;
+use Cake\Core\Plugin;
```
Comment (dereuromark) [inline tests/TestCase/Auth/TinyAuthorizeTest.php]: Order matters.

File: tests/TestCase/Auth/TinyAuthorizeTest.php [hunk 4]
```diff
@@ -1640,7 +1549,6 @@ public function testUserRolesUserWithoutPivotRoles() {
 		$res = $method->invokeArgs($object, [$user]);
 		$method->invoke($object);
 	}
-
 }
 
 class TestTinyAuthorize extends TinyAuthorize {
```
Comment (dereuromark) [inline tests/TestCase/Auth/TinyAuthorizeTest.php]: Ideally we also move this to its own file in TestApp.
But I can do that afterwards.

**Comment by dereuromark** [comment]
Use -f for sniffer to autofix those issues.

**Comment by dereuromark** [comment]
Way cleaner than before, thanks again!

**Pull request merged**